src/monitor/commands/end-monitoring.command.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/database/prisma.service';
+import { MonitorStatus } from '../utils/monitor.enum';
+import { InvalidMonitoringStatusException, MonitoringNotFoundException, NotTheResponsibleProfessorException } from '../utils/exceptions';
+
+@Injectable()
+export class EndMonitoringCommand {
+  constructor(private prisma: PrismaService) {}
+
+  async execute(id_monitoring: number, userId: number): Promise<void> {
```

Comment:
O ideal é a gente padronizar o case que a gente usa no repositório inteiro. Vamo sempre dar prioridade pro camelCase:

```suggestion
  async execute(monitorId: number, userId: number): Promise<void> {
```